src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTargetThemeHelper.java

```diff
@@ -69,12 +70,23 @@ public HandlerRegistration addEditorThemeStyleChangedHandler(
   
  
    private void syncToEditorTheme(TextEditingTarget editingTarget)
+   {
+      // delay execution so that the browser has a chance to apply styles
```

Comment:
I suppose we don't have a way to do this after some callback so that we're sure the styles are applied ?

Reply:
That's what I was hoping for, but as far as I know browsers don't have any way of notifying you when they've finished applying newly-inserted CSS styles.

In theory, there are some things we can do to force style / layout to be computed synchronously, e.g.

https://gist.github.com/paulirish/5d52fb081b3570c81e3a

but I didn't have any success with these.

Stylesheets do provide an `onload` event handler, but apparently that's fired just after the sheet is loaded but before the styles are actually applied:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#stylesheet_load_events